programs/locked-voter/src/instructions/open_dispute.rs

```diff
@@ -0,0 +1,84 @@
+use crate::*;
+/// Accounts for [voter::create_move_request].
+#[derive(Accounts)]
+pub struct OpenDispute<'info> {
+    /// [Escrow].
+    #[account(
+        mut,
+        has_one = owner,
+        has_one = locker
+    )]
+    pub escrow: Box<Account<'info, Escrow>>,
+    pub locker: Box<Account<'info, Locker>>,
+    /// [Recovery_Token_Account]
+    pub recovery_key: Signer<'info>,
+
+    /// [Request].
+    #[account(
+        init_if_needed,
+        seeds = [
+            b"Request".as_ref(),
+            escrow.key().as_ref(),
+            recovery_key.key().as_ref(),
+        ],
+        bump,
+        payer = feepayer,
+        space = MoveRequest::LEN
+    )]
+    pub request: Box<Account<'info, MoveRequest>>,
+
+    /// Authority of the [Old_Escrow].
+    pub owner: Signer<'info>,
+
+    /// Fee payer, and also new owner
+    #[account(mut)]
+    pub feepayer: Signer<'info>,
+
+    pub system_program: Program<'info, System>,
+}
+
+impl<'info> OpenDispute<'info> {
+    pub fn open_dispute(&mut self) -> Result<()> {
+        self.request.escrow = self.escrow.key();
+        self.request.recovery_key = self.feepayer.key();
+
+        let now = Clock::get()?.unix_timestamp;
+        invariant!(
+            now > self.escrow.freeze_timestamp,
+            "Error, account is frozen in the future"
+        );
+        self.escrow.freeze_timestamp = now;
```

Comment:
hacker can continuously call this function, so real user can never recover_fund, caused this field is overridden